braininventory/get.py

```diff
@@ -70,6 +70,13 @@ def __get_technique(df):
 
 def __get_locations(df):
     return df['locations'].value_counts().to_dict()
+	
+def __get_contributors(df): 
+"""
```

Comment:
* Wrong indentation.
* Sentences must be complete and begin with a capital letter.